README.md

```diff
@@ -68,6 +68,14 @@ export interface JSONScanner {
      * The length of the last read token.
      */
     getTokenLength(): number;
+    /**
+     * The zero-based start line number of the last read token.
+     */
+    getTokenLine(): number;
```

Comment:
Let's call this getTokenStartLine/StartCharacter, in case we ever decide to also provide endLine/endCharacter